pages/create-list.tsx

```diff
@@ -33,44 +38,108 @@ export const CreateListPage: NextPage = () => {
+    GroupBase<{
+      value: number
+      label: string
+    }>
+  > = {
+    option: (provided: CSSObjectWithLabel) => ({
+      ...provided,
+      minHeight: 36,
+    }),
+    control: (provided: CSSObjectWithLabel) => ({
+      ...provided,
+      width: '90px',
+      borderRadius: '28px',
+    }),
+    singleValue: (provided: CSSObjectWithLabel) => {
+      return { ...provided }
+    },
+  }
+
   return (
     <CreateListProvider value={handler}>
-      <Stack spacing="32px">
-        <Heading fontSize="heading1">Create a resource list</Heading>
+      <Stack spacing="16px" px={24}>
+        <Heading fontSize="heading1" mt={4}>
+          Create a resource list
+        </Heading>
         <Stack spacing="16px">
           <SearchBar
             baseData={baseServices}
             setData={setServices}
             searchFields={['name', 'description']}
           />
-          <HStack spacing="16px">
-            <SortMenu
-              data={services}
-              setData={setServices}
-              sortFieldsTextToVal={sortFieldsTextToVal}
-            />
-            <Select
-              isMulti
-              isSearchable
-              instanceId="taxonomySelect"
-              closeMenuOnSelect={false}
-              options={taxonomyOptions}
-              placeholder="Filter by resource category"
-              onChange={(e) => setFilters(e.map((e) => e.value))}
-            />
+          <HStack justifyContent={'space-between'}>
+            <HStack spacing="16px">
+              <SortMenu
+                data={services}
+                setData={setServices}
+                sortFieldsTextToVal={sortFieldsTextToVal}
+              />
+              <Select
+                isMulti
+                isSearchable
+                instanceId="taxonomySelect"
+                closeMenuOnSelect={false}
+                options={taxonomyOptions}
+                placeholder="Filter by resource category"
+                onChange={(e) => setFilters(e.map((e) => e.value))}
+              />
+              <Select
+                isSearchable
+                closeMenuOnSelect={true}
+                placeholder={`${maxAmountDisplayed}`}
+                options={displayAmountOptions}
+                onChange={(e) => {
+                  e ? setMaxAmountDisplayed(e.value) : null
+                }}
+                styles={pageViewStyles}
+              />
+            </HStack>
+            <Button w="fit-content" colorScheme="teal" onClick={onDrawerOpen}>
+              View your list
+            </Button>
           </HStack>
-          <Button w="fit-content" onClick={onDrawerOpen}>
-            View your list
-          </Button>
         </Stack>
-        {services.map((service) => (
-          <ServiceItem
-            key={service.id}
-            service={service}
-            onAlertOpen={onAlertOpen}
-            setSelectedService={setSelectedService}
-          />
-        ))}
+        <PaginatedList
```

Comment:
I think I'm gonna create our own hook for pagination so we can have greater control over the styling of components related to pagination but fine with using this component for now.